litmus/PTX/Proxy-Const-CoRR-cause2.litmus

```diff
@@ -1,5 +1,6 @@
-PTX fence
-"Two thread example with a constant proxy fence"
+PTX Proxy-Const-CoRR-cause2
```

Comment:
This is the test I referred in my previous comment. I find the comments confusing. I think you should follow the comment in `Proxy-Const-CoRR-cause-different-cta` and say that they sync because there is a fence between the store and the load (in both examples the fence is there, it just changes in which part the fence is)

Reply:
Comment updated